workspace.json

```diff
@@ -331,7 +331,8 @@
               "cd apps/frontend && mkdir -p public/build && $(npm bin)/webpack --node-env test",
               "mkdir -p dist/apps",
               "rm -r dist/apps/frontend || true",
-              "cp apps/frontend/public/ -r dist/apps/frontend"
+              "cp apps/frontend/public/ -r dist/apps/frontend",
+              "rm dist/apps/frontend/.gitignore"
```

Comment:
why rm the gitignore?

Reply:
It needs to be removed so that the dist directory does not ignore the build when copying over to push up to the remote repo. The whole thing is still gitignored though in the repo

Reply:
Oh I see